apps/prairielearn/src/sync/course-db.ts

```diff
@@ -1049,6 +1051,24 @@ async function validateQuestion(
     }
   }
 
+  if ('sharedPublicly' in question) {
+    if ('sharePublicly' in question) {
+      warnings.push('"sharedPublicly" is deprecated; use "sharePublicly" instead.');
+    } else {
+      errors.push('Cannot specify both "sharedPublicly" and "sharePublicly" in one question.');
+    }
+  }
+
+  if ('sharedSourcePublicly' in question) {
+    if ('shareSourcePublicly' in question) {
+      warnings.push('"sharedSourcePublicly" is deprecated; use "shareSourcePublicly" instead.');
+    } else {
+      errors.push(
+        'Cannot specify both "sharedSourcePublicly" and "shareSourcePublicly" in one question.',
+      );
+    }
```

Comment:
are the inner if-else statements backwards here? 

Should be an error if the inner if statement is also true, and a warning if the inner one isn't also true

Reply:
Yep.